[2025-05-26T08:32:29.398+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-26T08:32:29.409+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: x_videos_scraper_dag.x_login_task manual__2025-05-26T08:32:28.881826+00:00 [queued]>
[2025-05-26T08:32:29.415+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: x_videos_scraper_dag.x_login_task manual__2025-05-26T08:32:28.881826+00:00 [queued]>
[2025-05-26T08:32:29.415+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2025-05-26T08:32:29.423+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): x_login_task> on 2025-05-26 08:32:28.881826+00:00
[2025-05-26T08:32:29.426+0000] {standard_task_runner.py:72} INFO - Started process 49103 to run task
[2025-05-26T08:32:29.428+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'x_videos_scraper_dag', 'x_login_task', 'manual__2025-05-26T08:32:28.881826+00:00', '--job-id', '6783', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_x.py', '--cfg-path', '/tmp/tmpd2yui538']
[2025-05-26T08:32:29.430+0000] {standard_task_runner.py:105} INFO - Job 6783: Subtask x_login_task
[2025-05-26T08:32:29.466+0000] {task_command.py:467} INFO - Running <TaskInstance: x_videos_scraper_dag.x_login_task manual__2025-05-26T08:32:28.881826+00:00 [running]> on host f541e6bad3a5
[2025-05-26T08:32:29.522+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='x_videos_scraper_dag' AIRFLOW_CTX_TASK_ID='x_login_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-26T08:32:28.881826+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-26T08:32:28.881826+00:00'
[2025-05-26T08:32:29.523+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-26T08:32:37.099+0000] {logging_mixin.py:190} INFO - [INFO] Final URL: https://x.com/home
[2025-05-26T08:32:37.205+0000] {logging_mixin.py:190} INFO - [INFO] Storage state saved successfully.
[2025-05-26T08:32:37.297+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-26T08:32:37.305+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-26T08:32:37.306+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=x_videos_scraper_dag, task_id=x_login_task, run_id=manual__2025-05-26T08:32:28.881826+00:00, execution_date=20250526T083228, start_date=20250526T083229, end_date=20250526T083237
[2025-05-26T08:32:37.306+0000] {taskinstance.py:1554} INFO - Executing <lambda> callback
[2025-05-26T08:32:37.306+0000] {pipeline_x.py:39} INFO - DAG runs successfully
[2025-05-26T08:32:37.333+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-05-26T08:32:37.352+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-26T08:32:37.353+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
